runtime/queries/lean/injections.scm

```diff
@@ -0,0 +1,2 @@
+((comment) @markdown
+  (#offset! @markdown 0 3 0 -2))
```

Comment:
I believe `#offset!` is not a built-in (I think those are defined [here](https://github.com/tree-sitter/tree-sitter/blob/9df064c9fe31d35dc48244a6675588aee2d61a41/lib/binding_rust/lib.rs#L1385-L1484)), it's neovim specific.  Markdown injections may work without any sort of offset though. Also the way neovim does injections is pretty different from other tree-sitter based tools. To work with helix, this will need to become:

```suggestion
((comment) @injection.content
 (#set! injection.language "markdown"))
```